### PR TITLE
feat(database-change): database change task adapt streaming read sql file

### PR DESCRIPTION
**Pull request by smallsheeeep**
#### What type of PR is this?
In 4.2.4, we use streaming read sql content in database change task for lower memory cost. But the task in new task framework has not adapt it. This PR just fix it.
By the way, this PR add a new workflow for test. The workflow will only build ODC docker image using tag `latest` when you initiating a pull-request.

#### Which issue(s) this PR fixes:

Fixes #

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: .github/workflows/build_image_for_test.yaml [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+          docker build -t docker.io/oceanbase/odc:${odc_docker_image_tag} -f odc/Dockerfile .
+          docker save -o resources/odc-${odc_docker_image_tag}.tar.gz docker.io/oceanbase/odc:${odc_docker_image_tag}
+          popd
+      - name: Push docker image (x86_64)
+        run: |
+          docker login -p${{ secrets.DOCKER_PASSWORD }} -u${{ secrets.DOCKER_USERNAME }}
+          docker load -i distribution/docker/resources/odc-${odc_docker_image_tag}.tar.gz
+          docker tag oceanbase/odc:${odc_docker_image_tag} ${{ secrets.DOCKER_USERNAME }}/odc:${odc_docker_image_tag}
+          docker push ${{ secrets.DOCKER_USERNAME }}/odc:${odc_docker_image_tag}
+      - name: Upload docker image (x86_64)
```
Comment (yhilmare) [inline .github/workflows/build_image_for_test.yaml:132]: not necessary

File: .github/workflows/build_image_for_test.yaml [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+        uses: actions/setup-java@v3
+        with:
+          java-version: "8"
+          distribution: "temurin"
+          cache: maven
+      - name: Setup node 16
+        uses: actions/setup-node@v3
+        with:
+          node-version: "16"
+      - name: Build front static resources
```
Comment (yhilmare) [inline .github/workflows/build_image_for_test.yaml:55]: not necessary

File: .github/workflows/build_image_for_test.yaml (hunk 1 above)
Comment (smallsheeeep) [reply .github/workflows/build_image_for_test.yaml]: deleted.

File: .github/workflows/build_image_for_test.yaml (hunk 2 above)
Comment (smallsheeeep) [reply .github/workflows/build_image_for_test.yaml]: deleted.

**Review by krihy — APPROVED** [review]
LGTM

**Review by yizhouxw — APPROVED** [review]

**Pull request merged**